src/main/java/uk/gov/hmcts/reform/civil/service/search/CoscApplicationSearchService.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.civil.service.search;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.model.search.Query;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static org.elasticsearch.index.query.QueryBuilders.boolQuery;
+import static org.elasticsearch.index.query.QueryBuilders.matchQuery;
+import static uk.gov.hmcts.reform.civil.enums.CaseState.All_FINAL_ORDERS_ISSUED;
+
+@Service
+public class CoscApplicationSearchService extends ElasticSearchService {
+
+    public CoscApplicationSearchService(CoreCaseDataService coreCaseDataService) {
+        super(coreCaseDataService);
+    }
+
+    public Query query(int startIndex) {
+        return new Query(
+            boolQuery()
+                .must(matchQuery("data.coSCApplicationStatus", "Active"))
```

Comment:
TODO: Confirm query parameters!!!

Reply:
I think we also need to consider coscShedulerDeadline. .coSCApplicationPaidInFull And state is not required. But we can discuss about state